web/src/elements/ak-dual-select/ak-dual-select-provider.ts

```diff
@@ -72,16 +70,14 @@ export class AkDualSelectProvider extends CustomListenerElement(AKElement) {
 
     willUpdate(changedProperties: PropertyValues<this>) {
         if (changedProperties.has("searchDelay")) {
-            this.doSearch = debounce(this.doSearch.bind(this), this.searchDelay);
+            this.doSearch = debounce(
```

Comment:
Although it's unlikely the searchDelay will change multiple times during the course of *one* instance of this component's lifecycle, it's still bad practice to create the possibility of multiple wrappers.  Always start from the prototype in these cases!